### PR TITLE
doc(object-storage): add sections on configuring secrets for the helm chart

### PR DESCRIPTION
**Pull request by arschles**
Fixes https://github.com/deis/workflow/issues/101

Ref https://github.com/deis/deis/issues/4962

### CODE REVIEW

File: src/installing-workflow/configuring-object-storage.md [hunk 1]
```diff
@@ -51,6 +51,10 @@ The builder reads credentials from the below locations on the filesystem.
 - Key: `/var/run/secrets/object/store/access-key-id`
 - Secret `/var/run/secrets/object/store/access-key-secret`
 
+### Helm Chart
+
+If you are using the [Helm Chart for Workflow][helm-chart], put your credentials in the [`minio-user` secret][minio-user-secret] (under `access-key-id` and `access-secret-key`) before you `helm install`. For more information, see the [installation instructions][helm-install] for more details on using Helm.
```
Comment (bacongobbler) [inline src/installing-workflow/configuring-object-storage.md]: we should make a note here that the access key and secret **must** be `base64` encoded.

Comment (bacongobbler) [inline src/installing-workflow/configuring-object-storage.md]: not sure if this is totally necessary though... Seems like this information would be repeated in multiple locations.

Comment (arschles) [inline src/installing-workflow/configuring-object-storage.md]: I think it's worth putting it everywhere. I've made the mistake of _not_ encoding it many, many times

Comment (arschles) [inline src/installing-workflow/configuring-object-storage.md]: Added the notes about base64 encoding in bc58dc4e67ae9b110a1be96833c26a71489e10c3

**Pull request merged**